admin/main.js

```diff
@@ -10,6 +10,13 @@ jQuery.fn.addChild = function(html = "<div class='oscDiv'>")
 //sets which mixer type to get info for
 let mixer = "x32";
 
+//store current list of folders
+let folders = [];
+
+//Store universe data
+let firstUniverse = 1;
```

Comment:
Is this needed @cherry-john  - can't see where it's used

Reply:
Used to set bounds on old LX cards, Is legacy code that I moved to the top

Reply:
Ahh awesome